web/cobrands/shropshire/assets.js

```diff
@@ -45,16 +45,35 @@ fixmystreet.assets.add(defaults, {
     stylemap: new OpenLayers.StyleMap({
          'default': highways_style
      }),
-    always_visible: true,
-    non_interactive: true,
     usrn: {
         attribute: 'USRN',
         field: 'site_code'
     },
     road: true,
     asset_item: 'road',
     asset_type: 'road',
+    no_asset_msg_id: '#js-not-a-road',
```

Comment:
I think (though I'm not entirely sure) you need a no_asset_msgs_class as well - see e.g. equivalent code in Bucks assets.js and roads_message.html (and .com template too) - this is so if you click from a private road to a public road, the message gets correctly removed (otherwise I think it would only remove the "not on a road" message).